routers/books.js

```diff
@@ -103,14 +106,29 @@ bookRouter.put('/', (req, res) => {
  * @apiGroup Books
  * @apiVersion 0.1.0
  *
- *@apiParam {String} id Id of the Book being fetched.
+ *@apiParam {String} id Id of the Book being deleted.
  *
  * @apiSuccessExample {json} Success-Response:
- * 		HTTP/1.1 204 OK
+ * 		HTTP/1.1 204 No Content
  */
-bookRouter.delete('/', (req, res) => {
-	console.log('DELETE');
-	res.send('Got a DELETE request at /user');
+bookRouter.delete('/:id', [param('id').exists()], async (req, res) => {
```

Comment:
here you need to check that id is a valid mongoid,
if we use this route like this, i think an injection could be an option for an attacker,
because we just check for existence,
theres a "ismongoid" validator!